docs/pages/desktop-access/getting-started.mdx

```diff
@@ -27,104 +25,123 @@ This guide requires you to have:
-be used later for the PowerShell variable `$TeleportLDAPGroupName`.
+# Gives Teleport the ability to create LDAP containers in the CDP container.
+dsacls "CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CC;container;"
+# Gives Teleport the ability to create and delete cRLDistributionPoint objects in the CDP/Teleport container.
+dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;cRLDistributionPoint;"
+# Gives Teleport the ability to write the certificateRevocationList property in the CDP/Teleport container.
+dsacls "CN=TELEPORT,CN=CDP,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN " /I:T /G "$($SamAccountName):WP;certificateRevocationList;"
+# Gives Teleport the ability to create and delete certificationAuthority objects in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):CCDC;certificationAuthority;"
+# Gives Teleport the ability to write the cACertificate property in the NTAuthCertificates container.
+dsacls "CN=NTAuthCertificates,CN=Public Key Services,CN=Services,CN=Configuration,$DomainDN" /I:T /G "$($SamAccountName):WP;cACertificate;"
+
+```
 
-Finally, add the User to the Group you just made by right clicking the User and selecting `Add to group`. Type the name of the Group, and confirm correctness
-by clicking the “Check Names” button, then click “OK”. You should see a dialog saying “The Add to Group operation was successfully completed.”
+## Step 2/6. Prevent the service account from performing interactive logins.
 
-### Create and Apply Group Policy Object
+The Teleport service account is only needed to authenticate over LDAP, meaning that it needn't be able to login to windows machines like an ordinary user.
+Restrict it from doing so by creating a new Group Policy Object (GPO) linked to your entire domain, and then deny it interactive login.
 
-Next, open the "Start" menu and run "Group Policy Management". On the left pane, navigate to `$FOREST > Domains > $DOMAIN`,
-selecting your forest and domain names respectively. Right click `$DOMAIN` and select `Create a GPO in this domain, and Link it here...` from the context menu. 
-Give your GPO a name like `svc-teleport-gpo` and ensure "(none)" is selected for "Source Starter GPO", then hit "OK".
+### Create a GPO
+
+1. Open the `Start` menu and run `Group Policy Management`.
+2. On the left pane, navigate to `$FOREST > Domains > $DOMAIN`, selecting your forest and domain names respectively.
+3. Right click on `$DOMAIN` and select `Create a GPO in this domain, and Link it here...` from the context menu.
+4. Give your GPO a name like `Block Interactive Login` and ensure `(none)` is selected for `Source Starter GPO`, then hit `OK`.
```

Comment:
`Block Interactive Login` might be too vague, as there's nothing that indicates this is specific to the Teleport service account.

Reply:
How about `Block teleport-svc Interactive Login`

Reply:
Works for me!